newsletter-september-2021.rst

```diff
@@ -0,0 +1,101 @@
+.. post:: September 2, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in September.
+
+Read the Docs newsletter - September 2021
+=========================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We have finally published
+  :doc:`the first release candidate of version 1.0.0 of our Sphinx theme </theme-release-100rc1>`,
+  which adds support for recent versions of Sphinx and docutils among other things,
+  and announced our future plans for it.
+  Check out the linked blog post to know more.
+- The :doc:`first part of the new Read the Docs tutorial <readthedocs:tutorial/index>`,
```

Comment:
These both feel more like features to me (eg. `major features`), instead of company updates (eg. staffing or similar). Not sure what to do about it tho, but it makes our new features section feel really weak.

Reply:
Yes, the concept of "highlights" is a bit broad. Since the first one has a related blog post and is quite important and the second is grant work, and in the absence of a better suggestion, I'll leave them like this.